Automattic/wp-calypso
# Replace plansBannerUpsells test group with a separate A/B tests

This PR replaces test group `plansBannerUpsells` of `nudgeAPalooza` with a separate A/B tests. Instead of allocating entire traffic to a test group, this new test only allocates user who click on the sidebar nudge.

**Test plan:**

1. Choose a free site in Calypso
1. Refresh and confirm in AB test picker you are not assigned to any group of `plansBannerUpsells` test
1. Click on the "Free domain with a plan" sidebar nudge, refresh the page, and confirm you were assigned to any group of `plansBannerUpsells` test
1. Assign yourself to a "test" group
1. Click on the "Free domain with a plan" nudge and confirm you were redirected to `/plans?discount=free_domain` and that "Get a free domain name by upgrading to any plan listed below!" is visible above plans
1. Assign yourself to a "control" group
1. Click on the "Free domain with a plan" nudge and confirm you were redirected to `/plans` and that there is no additional message above plans
